paymentsheet-example/src/main/java/com/stripe/android/paymentsheet/example/playground/activity/AppearanceBottomSheetDialogFragment.kt

```diff
@@ -151,6 +185,12 @@ private fun AppearancePicker(
                     updateAppearance = updateAppearance,
                 )
             }
+            CustomizationCard("Embedded") {
```

Comment:
Should we conditionally show this section, only if embedded is enabled? That's what we do for all the other stuff. But probably not a big deal.